src/types.js

```diff
@@ -49,6 +49,18 @@ export type InputSelectionType = {
 
 export type Account = Auth;
 
+/**
+ * A list item for all private message list
+ */
+
+export type PmConversationListItem = {
```

Comment:
It's best to put a definition near the other definitions that are similar, or related.

Here, `UnreadStreamData` looks like a good candidate.